CMakeLists.txt

```
@@ -1417,6 +1417,14 @@ if(CONFIG_BUILD_OUTPUT_ADJUST_LMA)
     )
 endif()
 
+list(APPEND remove_sections ".comment" "COMMON")
+if(NOT CONFIG_EXCEPTIONS)
+  list(APPEND remove_sections ".eh_frame")
+endif()
+foreach(section ${remove_sections})
+  list(APPEND remove_sections_command $<TARGET_PROPERTY:bintools,elfconvert_flag_section_remove>${section})
```

This is not a command `remove_sections_command`, it's an argument, so I propose to change name into `remove_sections_argument_list`.

Also, please split the line.
```suggestion
  list(APPEND remove_sections_argument_list
       $<TARGET_PROPERTY:bintools,elfconvert_flag_section_remove>${section}
  )
```

---

@tejlmand thanks for your comments, I have updated this to fix them, could you help review it again? thanks.